websites/K/Kickassanime/dist/metadata.json

```diff
@@ -3,13 +3,16 @@
     "name": "Striker",
     "id": "215509157837537280"
   },
-  "service": "Kickassanime",
+  "service": "KickAssAnime",
```

Comment:
If you capitalize name etc you should do that  in folder name too 😄

Reply:
I changed it :P didn't have time to commit and check out tho